runtime-kernel/broadcom-wl/autobuild/prepare

```
@@ -0,0 +1,5 @@
+chmod -R go-w src/
+sed -i -e '/BRCM_WLAN_IFNAME/s/eth/wlan/' src/wl/sys/wl_linux.c
```

Please do not use `sed` to modify the package sources. Sed fails silently and do not always behave as we expect - please turn these into patches.

---

Converted to patches.